TSC-Charter.md

```diff
@@ -50,43 +67,54 @@ to continue attending TSC meetings as an observer.
+Planetary Software Organization, including:
+
+* Technical direction
+* Setting and maintaining governance rules for the conduct and make-up of the
+  TSC, Working Groups and other bodies within the TSC's domain
+* Creating new repositories and projects under the 
+  [_planetarysoftware_ GitHub organization](https://github.com/planetarysoftware)
+  as required
+* Maintaining a list of projects and working groups
+* Setting and maintaining appropriate standards for community discourse via the
+  various mediums under TSC control
+* Maintaining the list of additional Collaborators
+* Setting and maintaining standards covering contributions of code,
+  documentation and other materials
 * Mediating technical conflicts between Collaborators or Planetary Software Organization
-projects.
+projects
+
+The TSC serves as Planetary Software’s primary technical liaison
+body with external open source projects, consortiums, and groups.
+
+Many of these responsibilities will be delegated by the TSC to appropriate
+bodies such as the Working Groups.
 
-The TSC will define the Planetary Software Organization's release vehicles and serve as
-Planetary Software’s primary technical liaison body with external open
-source projects, consortiums and groups.
 
 ## Section 6. Planetary Software Organization Operations.
 
-The TSC will establish and maintain a development process for the Planetary Software Organization's Projects. The development process will establish guidelines
-for how the developers and community will operate. It will, for example,
-establish appropriate timelines for TSC review (e.g. agenda items must be
-published at least a certain number of hours in advance of a TSC
-meeting).
-
-There will be multiple Projects under the Planetary Software Organization organized by
-modules or subsystems. The TSC is responsible for organizing the Project
-structure, including possibly the creation and alignment of sub-Projects.
-Each Project must be within such policies as may be set by the TSC,
-have a well-defined scope and must work within that scope. The
-development process will provide for Projects to follow the lifecycle
-process as described in the [Project Lifecycle][] document. The development
-process will include a process for the TSC to oversee and approve changes
-in the lifecycle of a Project, which will include consideration of the
-following criteria:
+The TSC will establish and maintain a development process for the
+Planetary Software Organization's Projects. The development process
+will establish guidelines for how the developers and community will
+operate. It will, for example, establish appropriate timelines for
+TSC review (e.g. agenda items must be published at least a certain
+number of hours in advance of a TSC meeting).
+
+There will be multiple Top-Level Projects (TLPs) under the Planetary
+Software Organization as well as Top-Level Working Groups (TLWGs).
+The TSC is responsible for organizing the structure of these TLPs
```

Comment:
That sounds like a lot of control? What does it really mean?

Reply:
You're concerned about the 'responsible for organizing' phrase, that's a good point.  How about this sentence instead:

The TSC is responsible for helping these TLPs and TLWGs get organized.

Reply:
Will change.